etc/init.d/orchestrator.bash

```diff
@@ -54,8 +54,9 @@ start_daemon () {
 	post_start_daemon_hook 1>&2
 }
 
-# The file /etc/orchestrator_profile can be used to inject pre-service execution
+# This files can be used to inject pre-service execution
 # scripts, such as exporting variables or whatever. It's yours!
+[ -f /etc/default/orchestrator ] && . /etc/default/orchestrator
```

Comment:
Please explain why `/etc/default/orchestrator`? IF anything, I'd add `/etc/profile.d/orchestrator` which is what I should have done in the first place.

Reply:
/etc/default it's a default behavion on debian based distributives

Reply:
ok, i add usage  ```/etc/profile.d/orchestrator``` and ```/etc/default/orchestrator``` 

```/etc/profile.d/orchestrator``` will have major priority